src/plugins/vis_types/timeseries/server/lib/vis_data/request_processors/table/split_by_everything.ts

```diff
@@ -9,16 +9,25 @@
 import { buildEsQuery } from '@kbn/es-query';
 import { overwrite } from '../../helpers';
 
+import { isUIControlEnabled } from '../../../../../common/check_ui_restrictions';
+
 import type { TableRequestProcessorsFunction } from './types';
 
 export const splitByEverything: TableRequestProcessorsFunction =
-  ({ panel, esQueryConfig, seriesIndex }) =>
+  ({ panel, esQueryConfig, seriesIndex, capabilities }) =>
   (next) =>
   (doc) => {
     const indexPattern = seriesIndex.indexPattern || undefined;
 
     panel.series
-      .filter((c) => !(c.aggregate_by && c.aggregate_function))
+      .filter(
+        (c) =>
+          !(
+            c.aggregate_by &&
+            c.aggregate_function &&
+            isUIControlEnabled('aggregate_function', capabilities.uiRestrictions)
```

Comment:
Like in the other PR - I'm not a big fan of changing the behavior to make it work. It is really hard for the user to understand what's going on - can we resort to an error of some kind in this situation so the user can make an explicit change? It's possible the underlying data changes from normal to rollup without the user working in the editor which will change the behavior but still show data.